.github/workflows/osrm-backend.yml

```diff
@@ -155,7 +155,8 @@ jobs:
             CXXCOMPILER: g++-9
             CXXFLAGS: "-m32 -msse2 -mfpmath=sse"
             TARGET_ARCH: i686
-
+            ENABLE_CONAN: ON
```

Comment:
32-bit build was complaining about some linker errors related to TBB(even though there is 32-bit binary and I also tried to build it from source), so it is easier to simply use Conan instead which seems to work.